src/components/pages/AdminHome/AdminCourses.js

```diff
@@ -31,8 +31,8 @@ function AdminCourses(props) {
     <Layout>
       <AdminSidebar />
       <Content>
-        <div class="table-container">
-          <div class="header-container">
+        <div className="table-container">
+          <div className="header-container">
```

Comment:
Awesome job noticing to correct class to classname! Those little blips can be hard to identify.